resources/lib/SUBUtilities.py

```diff
@@ -260,3 +266,58 @@ def request(self, url, data=None, query_string=None, referrer=None, cookie=None)
             log("Failed to get url: %s\n%s" % (url, e))
             # Second parameter is the filename
         return content
+
+
+def title_from_focused_item(item_data):
+    label_type = xbmc.getInfoLabel("ListItem.DBTYPE")  # movie/tvshow/season/episode
+    label_movie_title = xbmc.getInfoLabel("ListItem.OriginalTitle")
+    is_movie = xbmc.getCondVisibility("Container.Content(movies)") or label_type == 'movie'
+    is_episode = xbmc.getCondVisibility("Container.Content(episodes)") or label_type == 'episode'
+
+    title = ''
+    if is_movie and label_movie_title and item_data['year']:
+        title = label_movie_title + " " + item_data['year']
+    elif is_episode and item_data['tvshow'] and item_data['season'] and item_data['episode']:
+        title = ("%s S%.2dE%.2d" % (item_data['tvshow'], int(item_data['season']), int(item_data['episode'])))
+
+    return title
+
+def getTVshowOriginalTitleByTMDBapi():
```

Comment:
Where is the check if it is not english?
Why add another request & all of this logic for users that have english titles?
What about "regular" users that has titles in Hebrew?

Reply:
How can I check if title is in English?
If it's possible I will add a condition here, but I don't know how.

It was tested with Hebrew and English titles.